docs/sources/panels/visualizations/pie-chart-panel.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Pie chart panel"
+keywords = ["grafana", "pie chart"]
+weight = 850
++++
+# Pie chart v2 panel
+> **Note:** This documentation refers to a beta feature.
+
+The pie chart compares series from one or more queries.
+
+The pie chart displays reduced series or values in a series as they relate to each other in the form of slices of a pie. The arc length, area and central angle of a slice are all proportional to the slices value, as it relates to the sum of all values. This type of chart is best used when you want a quick comparison of a small set of values in an aesthetically pleasing form.
+
+![Pie chart panel](/img/docs/pie-chart-panel/pie-chart-panel-7-5.png)
+
+## Data and field options
+
+Time series visualizations allow you to apply:
+
+- [Data transformations]({{< relref "../transformations/_index.md" >}})
+- [Field options and overrides]({{< relref "../field-options/_index.md" >}})
+- [Thresholds]({{< relref "../thresholds.md" >}})
+
+## Display options
+
+Use the following options to refine your visualization:
+
+### Show
+
+Choose how much information to show.
+
+- **Calculate -** Reduces a series to a single value to be displayed.
```

Comment:
I think I got this wrong. It reduces each series to a single value per series.

Reply:
Fixed